docs/source/strategies/wled.rst

```diff
@@ -4,15 +4,16 @@ WLED
 
 Supported domains: ``light``
 
-You can use WLED strategy for light strips which are controlled by [WLED](https://github.com/Aircoookie/WLED).
+You can use the manual WLED strategy for light strips which are controlled by `WLED <https://github.com/Aircoookie/WLED>`_.
 WLED calculates estimated current based on brightness levels and the microcontroller (ESP) used.
-Powercalc asks to input the voltage on which the lightstrip is running and optionally a power factor. Based on these factors the wattage is calculated.
+Powercalc asks to input the voltage on which the lightstrip is running and optionally a power factor. Based on these factors, the wattage is calculated.
 
 .. important::
     The brightness limiter must be turned on in WLED for this to work! Otherwise WLED will not provide an estimated current.
 
 You can setup sensors both with YAML or GUI.
-When you use the GUI select :guilabel:`wled` in the calculation_strategy dropdown.
+When you use the GUI, select the "Virtual power (manual)" sensor type, and :guilabel:`wled` in the "Calculation strategy" dropdown.
+If you have multiple segments, select any of them as the source entity; selecting a helper group with all segments will cause a failure to find the current sensor, even if entity IDs match perfectly.
```

Comment:
I don't think power sensor per segment will work correctly, as there is no `estimated_current` sensor supplied per segment by the WLED integration as far as I know.
So we might need to add a note that WLED strategy cannot be used for segments unfortunately.

Reply:
You may have misunderstood me. I agree it's not possible to get the current for each segment, but the overall current _is there_ in the WLED device. However, Powercalc doesn't find it for some reason, and the workaround right now is to select any of the segments instead - that's enough for Powercalc to grab the correct current sensor for the whole strip.

Reply:
Ah I understand what you mean now. Makes sense!